bin/push.sh

```diff
@@ -0,0 +1,4 @@
+git add .
```

Comment:
This file bin/push.sh can be removed since it overrides common git commands where more care should be taken to add particular files/changes instead of all the changes git add . as well as including a longer commit message.

Since that is the last file in the bin/ directory, the bin/ can be removed as well.

Reply:
Done